doc/release/1.14.0-notes.rst

```diff
@@ -253,37 +253,50 @@ enabling the new 1.13 "legacy" printing mode. This is enabled by calling
 
 In summary, the major changes are:
 
-* The ``repr`` of float arrays often omits a whitespace previously printed
-  in the sign position. See the new ``sign`` option to ``np.set_printoptions``.
-* Floating-point arrays and scalars use a new algorithm for decimal
-  representations, giving the shortest unique representation. This will
-  usually shorten ``float16`` fractional output, and sometimes ``float32`` and
-  ``float128`` output. ``float64`` should be unaffected.  See the new
-  ``floatmode`` option to ``np.set_printoptions``.
-* Float arrays printed in scientific notation no longer use fixed-precision,
-  and now instead show the shortest unique representation.
-* The ``str`` of floating-point scalars is no longer truncated in python2.
-* Non-finite complex scalars print like ``nanj`` instead of ``nan*j``.
+* To floating-point types:
+  * The ``repr`` of float arrays often omits a space previously printed
+    in the sign position. See the new ``sign`` option to ``np.set_printoptions``.
+  * Floating-point arrays and scalars use a new algorithm for decimal
+    representations, giving the shortest unique representation. This will
+    usually shorten ``float16`` fractional output, and sometimes ``float32`` and
+    ``float128`` output. ``float64`` should be unaffected.  See the new
+    ``floatmode`` option to ``np.set_printoptions``.
+  * Float arrays printed in scientific notation no longer use fixed-precision,
+    and now instead show the shortest unique representation.
+  * The ``str`` of floating-point scalars is no longer truncated in python2.
+* To other data types:
+  * Non-finite complex scalars print like ``nanj`` instead of ``nan*j``.
+  * ``NaT`` values in datetime arrays are now properly aligned.
+  * Arrays and scalars of ``np.void`` datatype are now printed using hex
+    notation.
+* To line-wrapping:
+  * The "dtype" part of ndarray reprs will now be printed on the next line
+    if there isn't space on the last line of array output.
+  * The ``linewidth`` format option is now always respected.
+    The `repr` or `str` of an array will never exceed this, unless a single
+    element is too wide.
+  * All but the last line of array strings will contain the same number of
+    elements.
+  * The last line of an array string will never have more elements than earlier
+    lines.
+* To summarization, the use of ``...`` to shorten long arrays:
```

Comment:
summarization <- summarize.

Reply:
I want the noun form here, since the other list items are nouns.

Reply:
Maybe "For summarization"

Reply:
"to noun" doesn't read right, "to nounize" does, even though "nounize" isn't a word ;)

Reply:
This was intended to be read as _"the changes are:"_ .. _"to summarization"_, which reads just fine. But perhaps that kind of continuation from list heading to item is confusing. I've replaced all of the _"to"_s with _"for"_s